crates/common/src/signer/loader.rs

```diff
@@ -105,9 +140,145 @@ fn load_secrets_and_keys(
     Ok(signers)
 }
 
+fn load_from_teku_format(
+    keys_path: String,
+    secrets_path: String,
+) -> eyre::Result<Vec<ConsensusSigner>> {
+    let entries = fs::read_dir(keys_path.clone())?;
+    let mut signers = Vec::new();
+
+    for entry in entries {
+        let entry = entry?;
+        let path = entry.path();
+
+        if path.is_dir() {
+            warn!("Path {path:?} is a dir");
+            continue;
+        }
+
+        let file_name = path
+            .file_name()
+            .and_then(OsStr::to_str)
+            .ok_or_eyre("File name not valid")?
+            .rsplit_once(".")
+            .ok_or_eyre("File doesn't have extension")?
+            .0;
+
+        match load_one(
+            format!("{keys_path}/{file_name}.json"),
+            format!("{secrets_path}/{file_name}.txt"),
+        ) {
+            Ok(signer) => signers.push(signer),
+            Err(e) => warn!("Sign load error: {e}"),
+        }
+    }
+
+    Ok(signers)
+}
+
+fn load_from_lodestar_format(
```

Comment:
I noticed that, but the [Lodestar docs](https://chainsafe.github.io/lodestar/run/validator-management/vc-configuration/#option-1-import-keys-to-lodestars-keystores-folder) seems different, since it requires a directory with keys an a file with a single password for all of them. Not sure which one do we want to support in this case.